tenderly/rpcProxy.ts

```diff
@@ -0,0 +1,61 @@
+import dotenv from 'dotenv'
+import type { Request } from 'express'
+import express from 'express'
+import { readFileSync } from 'fs'
+import { createProxyMiddleware, fixRequestBody } from 'http-proxy-middleware'
+import url from 'url'
+
+import { corsMiddleware } from './middleware/cors'
+import { createThrottleMiddleware } from './middleware/throttle'
+import { PROXY_ETHEREUM_NODE_PORT } from './utils/constants'
+import { getTenderlyRpcUrl } from './utils/getTenderlyRpcUrl'
+
+const { REACT_APP_ETHEREUM_NODE_URL } = dotenv.parse(readFileSync('.env.dev'))
+
+const port = Number(PROXY_ETHEREUM_NODE_PORT)
+const tenderlyRpcUrl = url.parse(getTenderlyRpcUrl())
+
+// only forward transaction calls to tenderly
+const tenderlyFilter = (_path: string, req: Request): boolean => {
+  const maybeEthMethod = req.body?.method
+  return (
+    typeof maybeEthMethod === 'string' &&
+    ['eth_sendTransaction', 'eth_signTransaction'].includes(maybeEthMethod)
+  )
+}
+
+const defaultFilter = (path: string, req: Request): boolean => {
+  return !tenderlyFilter(path, req)
+}
+
+const tenderlyProxy = createProxyMiddleware(tenderlyFilter, {
```

Comment:
Not sure if that's `createProxyMiddleware`, looks eval-y and noisy

<img width="869" alt="image" src="https://user-images.githubusercontent.com/17035424/235117974-1b1dc870-bb62-485f-b16d-313d49b9e1de.png">

Reply:
excellent point, solved good ser